eng/Version.Details.xml

```diff
@@ -15,6 +15,34 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>cf2938f44fc1494ae984c52a1568a03d51e513ca</Sha>
     </Dependency>
```

Comment:
Can a dependency by added for Microsoft.DotNet.Cli.CommandLine as well?  This is going to be needed for source-build.